Packs/MitreCaldera/Integrations/MitreCaldera/MitreCaldera.py

```diff
@@ -1323,6 +1323,132 @@ def get_objectives_command(client: Client, args: Dict[str, Any]) -> CommandResul
+    if client_name := params.get("client_name"):
+        filtered_operations = [operation for operation in filtered_operations if client_name in operation.get("name")]
+        
+    return filtered_operations
+
+
+def operation_to_incident(operation: dict, operation_date: str) -> dict:
+    """
+    Converts an operation dictionary into an incident dictionary.
+
+    Args:
+        operation (dict): A dictionary containing details of the operation.
+        operation_date (str): The date when the operation occurred.
+
+    Returns:
+        dict: A dictionary representing the incident, including the operation's ID, name, and the date it occurred.
+    """
+    operation_id = operation.get("id")
+    operation_name = operation.get("name")
+    incident = {
+        "name": f"Caldera: {operation_id} {operation_name}",
+        "occurred": f"{operation_date}",
+        "rawJSON": json.dumps(operation),
+    }
+    return incident
+
+
+def operations_to_incidents(operations: list, last_fetch_datetime: str) -> tuple[list, str]:
+    """
+    Converts a list of operations into a list of incidents and updates the latest incident time.
+
+    Args:
+        operations (list): A list of dictionaries, where each dictionary represents an operation.
+        last_fetch_datetime (str): The datetime string representing the last time incidents were fetched.
+
+    Returns:
+        tuple: A tuple containing:
+            - A list of dictionaries, each representing an incident.
+            - A string representing the latest incident time.
+    """
+    incidents: List[Dict[str, str]] = []
+    latest_incident_time = last_fetch_datetime
+    
+    # Parse last fetch datetime
+    parsed_last_fetch_datetime = dateparser.parse(last_fetch_datetime)
+    if not parsed_last_fetch_datetime:
+        raise ValueError(f"Failed parsing {last_fetch_datetime}")
+
+    for operation in operations:
+        operation_datetime = operation.get("start", "")
+        parsed_datetime = dateparser.parse(operation_datetime)
+        if parsed_datetime:
+            operation_datetime = parsed_datetime.isoformat()
+            demisto.debug(f"Original: {operation.get('start', '')}, ISO 8601: {operation_datetime}")
+        else:
+            demisto.debug(f"Failed to parse date: {operation_datetime}")
+        incident = operation_to_incident(operation, operation_datetime)
+        incidents.append(incident)
+
+        if parsed_datetime and parsed_datetime > parsed_last_fetch_datetime:
+            latest_incident_time = operation_datetime
+
+    return incidents, latest_incident_time
+
+
+def fetch_incidents(client: Client, args: Dict[str, Any], params: Dict[str, str]):
+    sort = args.get('sort')
+    last_run: Dict[str, str] = demisto.getLastRun()
+    demisto.debug(f"[Caldera] last run: {last_run}")
+
+    last_fetch = get_last_fetch_time(last_run, params)
+    demisto.debug(f"[Caldera] last fetch is: {last_fetch}")
+
+    operations = client.get_operations(sort, [], [])
```

Comment:
fetch incidents command doesn't have `sort` argument, what i meant is to call **client.get_operations** and pass the sort value hardcoded

Reply:
I had previously passed it as None, as we don't need a specific sort value. So what hardcoded value should be passed exactly ?